documentation/modules/proc-securing-kafka.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka.adoc
+
+[id='proc-securing-kafka-{context}']
+= Securing Kafka brokers
+
+This procedure shows the steps involved in securing Kafka brokers when running Strimzi.
+
+The security implemented for Kafka brokers must be compatible with the security implemented for the clients requiring access.
+
+* `Kafka.spec.kafka.listeners.*.authentication` matches `KafkaUser.spec.authentication matches`
+* `Kafka.spec.kafka.authorization` matches `KafkaUser.spec.authorization matches`
+
+The steps show the configuration for simple authorization and a listener using TLS authentication.
+For more information about listener configuration, see the xref:type-KafkaListeners-reference[`KafkaListeners` schema reference].
+
+Alternatively, you can use OAuth 2.0 for token-based xref:assembly-oauth-authentication_str[authentication] and xref:assembly-oauth-authorization_str[authorization],
```

Comment:
I guess this should mention also SCRAM-SHA-512 authentication as alternative.

Reply:
Yes. I've updated to make the links go back to concepts instead of separate procedures.

> Alternatively, you can use SCRAM-SHA or OAuth 2.0 for xref:con-securing-kafka-authentication-{context}[listener authentication], and OAuth 2.0 or OPA for xref:con-securing-kafka-authorization-{context}[Kafka authorization].